src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,71 @@
+package duke;
+
+import duke.task.Task;
+
+import java.io.Serializable;
+public class Ui implements Serializable{
+    String name = "Harry Potter";
+    String question = "Introducing the Wizarding World Organizer: Your Trusted Guide" +
+            " to Efficient & Effective Magical Planning";
+    String hello = "Hello muggle! I'm " + name + "\n" + question;
+    String bye = "\t" + "Expelliarmus! Hope to see you again muggle! :D";
+
+    public Ui() {
+    }
+
+    public void printHello() {
+        System.out.println(hello);
+    }
+
+    public void printBye() {
+        System.out.println(bye);
+    }
+
+    public void printDone(Task done) {
+        System.out.println("\t" + "Nice! I've marked this task " +
+                "as done:" + "\n" +
+                "\t " + done.taskString());
```

Comment:
Perhaps break should be before the operator, like the methods below

Reply:
Yup I see, thank you